lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java

```diff
@@ -142,23 +156,26 @@ private Object[] convertBirthDates(List<BirthDate> birthDates) {
     }
 
     private Object[] calculateEvidence(KBVItem kbvItem) {
-
         Evidence evidence = new Evidence();
         evidence.setTxn(kbvItem.getAuthRefNo());
 
-        EventProbe eventProbe = new EventProbe();
-
-        if (VC_THIRD_PARTY_KBV_CHECK_PASS.equalsIgnoreCase(kbvItem.getStatus())) {
+        if (VC_THIRD_PARTY_KBV_CHECK_NOT_AUTHENTICATED.equalsIgnoreCase(kbvItem.getStatus())) {
```

Comment:
it's makes sense to explicitly check for the scenario where there are sufficient questions & the user has answered too many incorrectly & only then set the V03 contraindicator. 
If the KbvItem / QuestionState doesn't have the results from the experian response they can be added so that you know how many questions were asked & answered incorrectly